python/pyspark/pandas/groupby.py

```diff
@@ -581,6 +581,56 @@ def mean(self, numeric_only: Optional[bool] = True) -> FrameLike:
             F.mean, accepted_spark_types=(NumericType,), bool_to_numeric=True
         )
 
+    # TODO: 'q' accepts list like type
+    def quantile(self, q: float = 0.5) -> FrameLike:
```

Comment:
I think we should add parameter `accuracy` like `dataframe.quantile` and `series.quantile`

Reply:
Yes, as offline discussion, the `quantile` is a little bit different than others, the distributed approximation, is different with pandas.

So just add `accuracy` as pandas-on-Spark special parameters and add a note.